molgenis-data-mysql/src/main/java/org/molgenis/data/importer/EmxImportService.java

```diff
@@ -49,12 +51,28 @@ public boolean canImport(File file, RepositoryCollection source)
 			{
 				if (entityName.equalsIgnoreCase(EmxMetaDataParser.ATTRIBUTES)) return true;
 				if (dataService.getMeta().getEntityMetaData(entityName) != null) return true;
+				if (canImportByHeuristic(entityName)) return true;
 			}
 		}
 
 		return false;
 	}
 
+	private boolean canImportByHeuristic(String entityName)
+	{
+		// entity is importable if entity name is the simple name of an existing entity,
+		// and only one entity with this simple name exists
+		List<EntityMetaData> entityMetaDatas = new ArrayList<EntityMetaData>();
```

Comment:
You can use the diamant operator here: List<EntityMetaData> entityMetaDatas = new ArrayList<>();